.github/CODEOWNERS

```diff
@@ -16,21 +16,21 @@ common/changes/@itwin/imodel-browser-react/ @raplemie
 packages/apps/storybook/src/imodel-browser/ @raplemie
 
 # delete-imodel
-packages/modules/delete-imodel/ @bentleyvk @veekeys
```

Comment:
☹️

Reply:
haha, wanna contribute? I guess I can leave you in :D